editor/animation_bezier_editor.cpp

```diff
@@ -834,16 +834,22 @@ void AnimationBezierTrackEdit::gui_input(const Ref<InputEvent> &p_event) {
 	}
 
 	if (p_event->is_pressed()) {
-		if (ED_GET_SHORTCUT("animation_editor/duplicate_selection")->matches_event(p_event)) {
+		if (ED_GET_SHORTCUT("animation_editor/duplicate_selected_keys")->matches_event(p_event)) {
```

Comment:
```suggestion
		if (ED_IS_SHORTCUT("animation_editor/duplicate_selected_keys", p_event)) {
```
Is the preferred syntax AFAIK, probably overlooked here at some point, can you confirm? Same for all in that case

Reply:
I see; all of animation_bezier_editor.cpp and animation_track_editor.cpp use `ED_GET_SHORTCUT(..)->matches_event()`. Should I just change this for the code that I touched, or should I change all of them, and if I should change all, should I add another commit for that, or just include in the last one?
Or maybe you might leave this for a future PR?

Reply:
I'd suggest changing them all, or doing them in a separate PR

Reply:
I'd opt for the separate PR then, since this one is changing a range of things already.